docs/others/upgrade-guide-from-v2-to-v3.md

```diff
@@ -0,0 +1,15 @@
+This is an upgrade guide to upgrade your Homie devices from v2 to v3.
+
+## New convention
+
+The Homie convention has been revised to v3 to be more extensible and introspectable. Be sure to [check it out](https://github.com/homieiot/convention/tree/v3.0.1).
+
+## API changes in the sketch
+
+1. Constructor of `HomieNode` needs third mandatory param `const char* type`:
+   E.g. `HomieNode lightNode("light", "Light");` -> `HomieNode lightNode("light", "Light", "switch");`.
+2. Signature of handleInput has changed to: `handleInput(const HomieRange& range, const String& property, const String& value)`
+   TODO: see Ping example.
+3. The `loop`s of HomieNodes could be run although there is no MQTT connection by setting param `setRunLoopDisconnected(true);`.
```

Comment:
Please note that this change is also in #develop (dev-branch of v2).

Reply:
Okay, shall we **move** it then in the v2 migration guide or **add** it there too?

Reply:
I think it should be added to a upgrade guide for 2.0 to 2.1 if we ever release a 2.1.

Reply:
point `3` is removed